cmd/boulder-wfe2/main.go

```diff
@@ -394,6 +407,12 @@ func main() {
 	logger.Infof("Server running, listening on %s....", c.WFE.ListenAddress)
 	handler := wfe.Handler(stats, c.OpenTelemetryHTTPConfig.Options()...)
 
+	if c.WFE.ListenAddress == "" {
```

Comment:
This block should come before before line 407. The log line at 414 is a duplicate of 407 and can be removed.

Reply:
Done